solution/ui/regulations/js/src/components/SupplementalContent.vue

```diff
@@ -13,6 +13,9 @@
         <h2 v-if="!requested_categories" id="subpart-resources-heading">
             {{ activePart }} Resources
         </h2>
+        <div class="resource_btn_container">
+            <a :href="resourceLink" class="search_resource_btn" >Search These Resources</a>
```

Comment:
this is where you could use the `default-btn` and `actio-btn` classes and then use `search-resource-btn` class (I think we should use kebab case for classes on the front end) to fine-tune the styling for this specific button.

Reply:
changed

Reply:
Closer.  We still need to have the text color be white in the hover state, and the left padding should be zero:

<img width="304" alt="Screen Shot 2022-08-15 at 11 32 52" src="https://user-images.githubusercontent.com/1651907/184666400-a0f37ea0-077e-406f-adce-caf1ed64d4e0.png">